### DISCUSSION TITLE
Multiple passes

### DISCUSSION THREAD

**Discussion by Iteilborg** [arkypita/LaserGRBL#1857] [General]
Just got started with my new laser, I'm having an issue with it burning at least two passes even though I have it set to only burn one. Am I missing a setting that I need to change? Or does it always just burn twice?

**Comment by Tribo-2dot0** [comment]
If next to the Start Work button (green triangle) it is set to 1 (1 pass), then you are probably having double lines.
What type of file is LaserGrbl loading?
